src/Paket.Core/Common/Constants.fs

```diff
@@ -114,12 +114,9 @@ let GitRepoCacheFolder = Path.Combine(LocalRootForTempData,".paket","git","db")
 let [<Literal>] GlobalPackagesFolderEnvironmentKey = "NUGET_PACKAGES"
```

Comment:
This gets referenced from https://github.com/fsprojects/Paket/blob/eed370a1ac41eab5096faae8b0621777c816b9e5/integrationtests/Paket.IntegrationTests/NugetDownloadSpecs.fs#L24 so I suppose removing that would be a breaking change